static/js/message_list.js

```diff
@@ -360,9 +360,7 @@ MessageList.prototype = {
         }
         var trailing_bookend_content, subscribed = stream_data.is_subscribed(stream);
         if (subscribed) {
-            if (this.last_message_historical) {
-                trailing_bookend_content = this.subscribed_bookend_content(stream);
-            }
+            trailing_bookend_content = this.subscribed_bookend_content(stream);
```

Comment:
At the end of a stream, not matter whether `this.last_message_historical` is true or false. If user is subscribed to stream, user can see a button to unsubscribe to stream(rather than first opening Streams page and then unsubscribing).